be/src/storage/lake/meta_file.cpp

```diff
@@ -125,11 +129,6 @@ void MetaFileBuilder::apply_opwrite(const TxnLogPB_OpWrite& op_write, const std:
         file_meta.set_name(orphan_file);
         _tablet_meta->mutable_orphan_files()->Add(std::move(file_meta));
     }
-    for (const auto& del_file : op_write.dels()) {
-        FileMetaPB file_meta;
-        file_meta.set_name(del_file);
-        _tablet_meta->mutable_orphan_files()->Add(std::move(file_meta));
-    }
```

Comment:
If you remove these codes, how will these files be garbage collected?

Reply:
It will be removed when ROWSET is compacted and removed, I add del files to ROWSET.

Reply:
You can check the modification of `be/src/storage/lake/vacuum.cpp`.